Source/WebCore/testing/Internals.h

```diff
@@ -59,6 +59,10 @@
 #include "AudioSession.h"
 #endif
 
+#if ENABLE(DAMAGE_TRACKING)
+#include "Damage.h"
+#endif
```

Comment:
`Damage.h` has a `#if ENABLE(DAMAGE_TRACKING)` guard internally so no need to guard here